src/rpp/rpp/operators/combine_latest.hpp

```diff
@@ -13,59 +13,36 @@
 #include <rpp/operators/fwd.hpp>
 
 #include <rpp/defs.hpp>
-#include <rpp/disposables/composite_disposable.hpp>
+#include <rpp/operators/details/combining_strategy.hpp>
 #include <rpp/operators/details/strategy.hpp>
-#include <rpp/operators/details/utils.hpp>
-#include <rpp/schedulers/current_thread.hpp>
-
-#include <memory>
 
 namespace rpp::operators::details
 {
 template<rpp::constraint::observer Observer, typename TSelector, rpp::constraint::decayed_type... Args>
-class combine_latest_disposable final : public composite_disposable
+class combine_latest_disposable final : public combining_disposable<Observer, Args...>
 {
 public:
     explicit combine_latest_disposable(Observer&& observer, const TSelector& selector)
-        : observer_with_mutex{std::move(observer)}
-        , selector{selector}
+        : combining_disposable<Observer, Args...>(std::forward<Observer>(observer))
```

Comment:
It should be move, not forward (same for zip)

Reply:
done